pages/settings/SettingsNode.qml

```diff
@@ -262,80 +262,94 @@ Rectangle{
             }
 
-            GridLayout {
-                columns: 2
-                columnSpacing: 32
-
-                MoneroComponents.LineEdit {
-                    id: daemonUsername
-                    Layout.fillWidth: true
-                    labelText: qsTr("Daemon username") + translationManager.emptyString
-                    text: persistentSettings.daemonUsername
-                    placeholderText: qsTr("(optional)") + translationManager.emptyString
-                    placeholderFontSize: 15
-                    labelFontSize: 14
-                    fontSize: 15
-                }
-
-                MoneroComponents.LineEdit {
-                    id: daemonPassword
-                    Layout.fillWidth: true
-                    labelText: qsTr("Daemon password") + translationManager.emptyString
-                    text: persistentSettings.daemonPassword
-                    placeholderText: qsTr("Password") + translationManager.emptyString
-                    password: true
-                    placeholderFontSize: 15
-                    labelFontSize: 14
-                    fontSize: 15
-                }
-            }
+            ColumnLayout {
+                spacing: 0
+
+                Repeater {
+                    model: remoteNodesModel
+
+                    Rectangle {
+                        height: 30
+                        Layout.fillWidth: true
+                        color: itemMouseArea.containsMouse || index === remoteNodesModel.selected ? MoneroComponents.Style.titleBarButtonHoverColor : "transparent"
+
+                        Rectangle {
+                            color: MoneroComponents.Style.appWindowBorderColor
+                            anchors.right: parent.right
+                            anchors.left: parent.left
+                            anchors.top: parent.top
+                            height: 1
+                            visible: index > 0
+
+                            MoneroEffects.ColorTransition {
+                                targetObj: parent
+                                blackColor: MoneroComponents.Style._b_appWindowBorderColor
+                                whiteColor: MoneroComponents.Style._w_appWindowBorderColor
+                            }
+                        }
 
-            MoneroComponents.CheckBox {
-                id: setTrustedDaemonCheckBox
-                checked: persistentSettings.is_trusted_daemon
-                onClicked: {
-                    persistentSettings.is_trusted_daemon = !persistentSettings.is_trusted_daemon
-                    currentWallet.setTrustedDaemon(persistentSettings.is_trusted_daemon)
-                }
-                text: qsTr("Mark as Trusted Daemon") + translationManager.emptyString
-            }
+                        Rectangle {
+                            anchors.fill: parent
+                            anchors.topMargin: 5
```

Comment:
```suggestion
```

Destroys the vertical text alignment, unless there is a reason for it I didn’t figure out.